src/manufacturer/manufacturerLandingPage.component.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+  Box,
+  Button,
+  Grid,
+  Typography,
+  Link as MuiLink,
+  LinearProgress,
+} from '@mui/material';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import { Link, useLocation } from 'react-router-dom';
+import { useManufacturer } from '../api/manufacturer';
+
+import ManufacturerDialog from './manufacturerDialog.component';
+import React from 'react';
+
+function ManufacturerLandingPage() {
+  const location = useLocation();
+
+  const manufacturerId = location.pathname.replace(
+    '/inventory-management-system/manufacturer/',
+    ''
+  );
+
+  const { data: manufacturerData, isLoading: manufacturerDataLoading } =
+    useManufacturer(manufacturerId);
+
+  const [editManufacturerDialogOpen, setEditManufacturerDialogOpen] =
+    React.useState<boolean>(false);
+
+  return (
+    <Grid container>
+      <Grid sx={{ padding: '8px' }} item>
+        <Button
+          component={Link}
+          to={`/inventory-management-system/manufacturer/`}
+          sx={{ margin: '8px' }}
+          variant="outlined"
+        >
+          {manufacturerData ? 'Manufacturer table view' : 'Home'}
+        </Button>
+        <Button
+          disabled={!manufacturerData}
+          sx={{ margin: '8px' }}
+          variant="outlined"
+          onClick={() => {
+            setEditManufacturerDialogOpen(true);
+          }}
+        >
+          Edit
+        </Button>
+      </Grid>
+      {manufacturerData && (
+        <Grid item xs={12}>
+          <Box
```

Comment:
This and the other boxes should probably be `Grid item`s, the one above would then need to be `Grid item container`, putting the `px: '192px'` in that should also avoid repeating it (like https://github.com/ral-facilities/inventory-management-system/blob/49c87c570038916d40083b49b5fa0b17c775e9d5/src/catalogue/items/catalogueItemsLandingPage.component.tsx#L115-L125)

I also think the URL/Telephone number headings should align with the address
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/a8f2dd1e-00eb-4574-ac27-18e8a063c5a2)